be/src/common/config.h

```diff
@@ -519,6 +519,8 @@ namespace config {
     CONF_Int64(brpc_max_body_size, "209715200");
     // Max unwritten bytes in each socket, if the limit is reached, Socket.Write fails with EOVERCROWDED
     CONF_Int64(brpc_socket_max_unwritten_bytes, "67108864");
+    // If batch size large than brpc_attachment_threashold, use attachment instead
```

Comment:
```suggestion
    // If batch size is larger than brpc_attachment_threshold, use attachment instead
```
I have a question, why not use threshold directly for all the packets?